py/dml/codegen.py

```diff
@@ -3031,12 +3031,19 @@ def stmt_select(stmt, location, scope):
                     else_dead = True
                     break
 
+            iterations = len(clauses)
             if else_dead:
                 (last_cond, last_stmt) = clauses.pop(-1)
                 assert last_cond.constant and last_cond.value
                 if_chain = last_stmt
             else:
                 if_chain = codegen_statement(else_ast, location, scope)
+            if iterations > WBIGUNROLL.limit and isinstance(lst, List):
+                report(WBIGUNROLL(stmt.site,
+                                  '#'*(stmt.site.dml_version() != (1, 2))
```

Comment:
space around binop

Reply:
This is a stylistic choice we can quibble about: when combining `+` and `*`, I actually prefer omitting spaces around `*`; I feel that increases readability re. operator precedence. I see it as similar to implicit multiplication in math via concatenation (`2(3)`).

Reply:
Here we don't combine `+` and `*`. but we do combine `*` and ` != `.

And I agree that PEP-8 is not always optimal; in particular, I find `f(foo=bar == 2)` a bit weird. But I just accept and obey the style for consistency, unless there's strong reasons not to.

Reply:
> Here we don't combine `+` and `*`

I mean... we are, though the `+` _is_ on a new line. That's good enough for readability, I admit. Though that doesn't apply to the below `'#'*(site.dml_version() != (1, 2)) + 'foreach',`

> we do combine `* and `!=`.

Not on the same syntactic level, the `!=` is within an `()`; I'm talking about `*` and `+`, which is on the same syntactic level, and where operator precedence determines how associations are made. This is where I'm arguing that omitting spaces around `*` such that such parts are more obviously whole terms helps readability. To exemplify, compare
```
   bla_bla_blau * blib * blub_blub + glabradahadaba + blara * dabara
+ canyoutellthat * ihavegivenup + on * variable * naming
```
vs.
```
   bla_bla_blau*blib*blub_blub + glabradahadaba + blara*dabara
+ canyoutellthat*ihavegivenupon + on*variable*naming
```

And that does apply to `'#'*(site.dml_version() != (1, 2)) + 'foreach'`, but not as strongly. If you write out:
```
'#' * (site.dml_version() != (1, 2)) + 'foreach'
```
It's not as obviously clear at a glance that the `(site.dml_version() != (1, 2))` is used for controlling a term of the string concatenation sequence, as opposed to a term in of itself. Of course, the counterargument to that would be parentheses...:
```
('#' * (site.dml_version() != (1, 2))) + 'foreach'
```
If you insist on spacing, I would do the above. even though i kinda hate it.